SetupDataPkg/Tools/VariableList.py

```diff
@@ -140,6 +168,14 @@ def binary_to_object(self, binary_representation):
     def size_in_bytes(self):
         return struct.calcsize(self.pack_format)
 
+    def check_bounds(self, value, min, max):
+        if value is not None and min is not None:
```

Comment:
For any of these except enum and bool, is it valid to have `None` for the min or max? We should have at least the type default, right? In which case, should we fail in the relevant check_bounds methods if min or max is None?

Reply:
Removed the extra checks